### PR TITLE
Fix: type was not setting correctly when creating

### PR DESCRIPTION
**Pull request by bogensberger**
a token filter with default name and no type

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: type was not setting correctly when creating a token filter
```
Comment (mikethebeer) [inline CHANGES.txt]: was not **set** correctly

**Comment by mikethebeer** [comment]
lgtm

File: sql/src/main/java/io/crate/analyze/CreateAnalyzerStatementAnalyzer.java [hunk 2]
```diff
@@ -179,6 +179,7 @@ public CreateAnalyzerAnalyzedStatement visitTokenFilters(TokenFilters tokenFilte
                         throw new IllegalArgumentException(String.format(Locale.ENGLISH,
                             "token-filter name '%s' is reserved, 'type' property forbidden here", name));
                     }
+                    properties.get().add(new GenericProperty("type", StringLiteral.fromObject(name)));
```
Comment (seut) [inline sql/src/main/java/io/crate/analyze/CreateAnalyzerStatementAnalyzer.java]: would make sense to store properties.get() in a local var

Comment (seut) [inline sql/src/main/java/io/crate/analyze/CreateAnalyzerStatementAnalyzer.java]: name is already declared as String, so pls use new StringLiteral(name) instead

**Comment by seut** [comment]
pls rebase

**Comment by seut** [comment]
lgtm

**Pull request merged**